<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EPuckUserInterface</class>
 <widget class="QWidget" name="EPuckUserInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TP10</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="motorGroup">
     <property name="title">
      <string>Motors</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSlider" name="motor0">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="motor1">
          <property name="minimum">
           <number>-1000</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBelow</enum>
          </property>
          <property name="tickInterval">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ledGroup">
     <property name="title">
      <string>LEDS</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="led0">
        <property name="text">
         <string>1</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="led1">
        <property name="text">
         <string>2</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="led2">
        <property name="text">
         <string>3</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="led3">
        <property name="text">
         <string>4</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="led4">
        <property name="text">
         <string>5</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="led5">
        <property name="text">
         <string>6</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="led6">
        <property name="text">
         <string>7</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="led7">
        <property name="text">
         <string>8</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="proxiGroup">
     <property name="title">
      <string>Proxi</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>4</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>5</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>6</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>7</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>8</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="proxi0">
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="proxi1">
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="proxi2">
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="proxi3">
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="proxi4">
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLabel" name="proxi5">
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QLabel" name="proxi6">
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="7">
       <widget class="QLabel" name="proxi7">
        <property name="text">
         <string>0000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="port">
       <property name="text">
        <string>/dev/rfcomm0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="quitButton">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>quitButton</sender>
   <signal>clicked()</signal>
   <receiver>EPuckUserInterface</receiver>
   <slot>quitButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>motor0</sender>
   <signal>valueChanged(int)</signal>
   <receiver>EPuckUserInterface</receiver>
   <slot>updateMotors(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>motor1</sender>
   <signal>valueChanged(int)</signal>
   <receiver>EPuckUserInterface</receiver>
   <slot>updateMotors(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>led0</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EPuckUserInterface</receiver>
   <slot>updateLeds(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>led1</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EPuckUserInterface</receiver>
   <slot>updateLeds(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>led2</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EPuckUserInterface</receiver>
   <slot>updateLeds(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>led3</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EPuckUserInterface</receiver>
   <slot>updateLeds(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>led4</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EPuckUserInterface</receiver>
   <slot>updateLeds(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>356</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>led5</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EPuckUserInterface</receiver>
   <slot>updateLeds(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>426</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>led6</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EPuckUserInterface</receiver>
   <slot>updateLeds(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>led7</sender>
   <signal>stateChanged(int)</signal>
   <receiver>EPuckUserInterface</receiver>
   <slot>updateLeds(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>connectButton</sender>
   <signal>clicked()</signal>
   <receiver>EPuckUserInterface</receiver>
   <slot>connectClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>quitButtonClicked()</slot>
  <slot>updateMotors(int)</slot>
  <slot>updateLeds(int)</slot>
  <slot>connectClicked()</slot>
 </slots>
</ui>
